### ISSUE TITLE
Decode data client side

### ISSUE THREAD

**Issue by xseignard** [moovel/node-yolo#24]
Hello again,

I'd like to send the `modified` image frame via websockets to a client.

For now all what I get is a green image on the client side, I assume I do something wrong, any idea on how to decode the frame client side? or server side  to then send it as a jpeg base64.

See my code here: https://github.com/xseignard/yolo9000-test/blob/master/src/public/main.js#L14-L26

Best regards

**Comment by OrKoN** [comment]
Hi @xseignard so far I have decoded with ffmpeg only https://github.com/moovel/node-yolo/blob/master/test/darknetImageTest.js#L12

It requires the pix_fmt set which basically defines the order of colors. For me, bgr24 is working but I have also experienced that different opencv versions have different default pixel format.

**Comment by OrKoN** [comment]
Please note that the image recognition won't work concurrently so you will need to make sure only one image is recognized at a time.

**Comment by xseignard** [comment]
Yes, it works server side for me too, and sending the image encoded as base 64 through websocket also works. But I'm pretty sure the decoding of the raw rgb24 data can be achieved client side, it will eliminate the burden of using ffmpeg on the server side.
I haven't found a solution so far. I'll investigate it further.

**Comment by OrKoN** [comment]
Hi @xseignard have you found a solution?
